burn-compute/src/tune/tune_cache.rs

```diff
@@ -20,24 +43,116 @@ pub enum TuneCacheResult<K> {
 
 impl<K: AutotuneKey> TuneCache<K> {
     pub(crate) fn new() -> Self {
-        TuneCache {
-            cache: HashMap::new(),
+        #[cfg(feature = "autotune-persistent-cache")]
+        {
+            let mut cache = TuneCache {
+                cache: HashMap::new(),
+                persistent_cache: HashMap::new(),
+            };
+            if let Err(e) = cache.load() {
+                eprintln!(
+                    "Unable to load autotune cache. Cache will be ignored ({}).",
+                    e
+                );
+            }
+            cache
+        }
+
+        #[cfg(not(feature = "autotune-persistent-cache"))]
+        {
+            TuneCache {
+                cache: HashMap::new(),
+            }
         }
     }
 
     #[allow(clippy::borrowed_box)]
```

Comment:
Is the clippy warning still an issue? I don't see any borrowed box here (only an owned box).